Tools/Scripts/webkitdirs.pm

```diff
@@ -1189,18 +1189,10 @@ sub productDir
 
 sub executableProductDir
 {
-    my $productDirectory = productDir();
-
-    my $binaryDirectory;
-    if (isAnyWindows() && !isPlayStation()) {
-        $binaryDirectory = "bin64";
-    } elsif (isGtk() || isJSCOnly() || isWPE() || isPlayStation()) {
-        $binaryDirectory = "bin";
-    } else {
-        return $productDirectory;
+    if (isGtk() || isJSCOnly() || isWPE() || isPlayStation() || isAnyWindows()) {
```

Comment:
There a better if statement here? Check if Apple?

Reply:
Fixed. Thank you.